downstream/titles/automation-mesh/master.adoc

```diff
@@ -9,12 +9,12 @@ include::attributes/attributes.adoc[]
 
 
 // Book Title
-= Red Hat Ansible Automation Platform Automation Mesh Guide for VM-based installations
+= Red Hat Ansible Automation Platform Automation mesh guide for VM-based installations
```

Comment:
I believe Automation should be lower case too, so:
Red Hat Ansible Automation Platform automation mesh guide for VM-based installations